client/src/themes/theme.js

```diff
@@ -2,13 +2,20 @@ import { createMuiTheme } from "@material-ui/core";
 
 export const theme = createMuiTheme({
   typography: {
-    fontFamily: '"Roboto"',
+    fontFamily: ["Proxima Nova", "Roboto"].join(","),
```

Comment:
```suggestion
    fontFamily: "'Proxima Nova', 'Roboto'"
```
This join operator isn't really necessary as the list of fonts is not created programmatically. As long as it's hardcoded like this you can just use a string

Reply:
the join was used for legibility in the code,

``` 
fontFamily: [
      "Proxima Nova",
      "Roboto"
    ].join(","),
```

but it seems prettier liked a single line.  Unless the list grows, it probably isn't necessary though...